apps/admin-portal/src/components/roles/role-list.tsx

```diff
@@ -37,21 +38,29 @@ export const RoleList: React.FunctionComponent<RoleListProps> = (props: RoleList
         handleRoleDelete
     } = props;
 
+    const handleRoleEdit = (roleId: string) => {
+        history.push(`roles/${ roleId }`);
```

Comment:
Shall we get the `roles` path from a constant

Reply:
Fixed